<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CES Device Simulator - Group 12</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>681</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="deviceFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>461</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 207)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="currentUpButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>380</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>▲</string>
      </property>
     </widget>
     <widget class="QPushButton" name="currentDownButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>380</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>▼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="countdownCycleButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Countdown Cycle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toggleFrequencyButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Toggle Frequency</string>
      </property>
     </widget>
     <widget class="QFrame" name="displayFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>321</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QProgressBar" name="batteryBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="format">
        <string>Battery: %p%</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="currentBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="format">
        <string>Current: %v µA</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="countdownCycleBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>240</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Cycle: 20 min</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="frequencyBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>240</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Frequency: 0.5 Hz</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="circuitBrokenWarning">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(239, 41, 41);</string>
       </property>
       <property name="text">
        <string>Circuit Broken!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLCDNumber" name="therapyCountDownDisplay">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>361</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="warningDisplay">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="togglePowerButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Toggle Power</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="waveformBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>280</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Waveform: Alpha</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="therapyActiveNotification">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(29, 108, 0);</string>
       </property>
       <property name="text">
        <string>Therapy Active</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="currentControlLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>340</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Control</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="toggleTherapyButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Therapy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toggleSaveButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>420</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Therapy Saving: ON</string>
      </property>
     </widget>
     <widget class="QComboBox" name="savedTherapyComboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="toggleWaveformButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>380</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Toggle Waveform</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="toggleCircuitButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>400</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle Circuit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleFaultButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>400</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Simulate Fault</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>440</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleFastforwardButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>440</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle Fastforward</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="debugPanel">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>251</width>
       <height>371</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Debug Panel:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCOMP3004_group12">
    <property name="title">
     <string>COMP3004_group12</string>
    </property>
   </widget>
   <addaction name="menuCOMP3004_group12"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
